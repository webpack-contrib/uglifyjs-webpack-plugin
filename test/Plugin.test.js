/* eslint-disable
  import/order
 */

import webpack from './helpers/compiler';
import UglifyJsPlugin from '../src/index';

describe('Plugin', () => {
  test('Defaults', async () => {
    const config = {
      plugins: [new UglifyJsPlugin()],
    };

    const stats = await webpack('entry.js', config);
    const { assets } = stats.compilation;

    const main = assets['main.bundle.js'].source();

    expect(main).toMatchSnapshot();
  });
});
