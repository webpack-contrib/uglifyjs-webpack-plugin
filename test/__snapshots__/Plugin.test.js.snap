// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plugin Defaults 1`] = `"(window.webpackJsonp=window.webpackJsonp||[]).push([[0],[function(o,n){o.exports=function(){console.log(7)}}],[[0,1,0]]]);"`;
